extensions/wikia/AdEngine/src/context.js

```diff
@@ -0,0 +1,50 @@
+export default {
+	adUnitId: '/{custom.dfpId}/wka.{targeting.s0}/{targeting.s1}//{targeting.s2}/{src}/{slotName}',
+	custom: {
+		adLayout: 'asd', // TODO: add dynamically
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app